lib/translator/translation.cc

```diff
@@ -41,6 +41,9 @@ BeginResponse Translation::Begin(absl::Span<const uint8_t> scsi_cmd,
   switch (opc) {
     case scsi::OpCode::kInquiry:
       break;
+    case scsi::OpCode::kVerify10:
```

Comment:
Do we care about Verify16?

Reply:
i don't think it will be a priority any time soon